src/HumanView.cpp

```diff
@@ -180,10 +184,10 @@ void HumanView::drawEndLevelDialogue() {
         switch (dialogueStage) {
             //eventually change to be appropriate based on score
             case 0: 
-                response = (logic.getScore(currentLevel) < logic.getGoalScore(currentLevel)) ? strings.getResponse("DateNo") : strings.getResponse("DateYes");
+                response = (logic.getPlayerFail()) ? strings.getResponse("DateNo") : strings.getResponse("DateYes");
```

Comment:
Something I probably should have realized earlier, but I never actually increase the levelsWon counter in HumanView. It would probably make sense to increase it here when we check whether the player has successfully completed the level or not. I am not sure where else we would do that.